.github/ISSUE_TEMPLATE/confiq.yml

```
@@ -0,0 +1,30 @@
+name: meta/repository Report
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
```

This is the title of the issue the user is making.

I like to put in a suggestion here which goes something like this:
```suggestion
title: "📇 <feature name>"
```